api/proxy/servers/rest/server.go

```diff
@@ -18,11 +18,18 @@ import (
 	"github.com/gorilla/mux"
 )
 
+// ProxyConfig ... ProxyConfig is used by the /config endpoint
+type ProxyConfig struct {
```

Comment:
not a fan of this name but not super sure what else to use. Maybe PubliclyExposedConfig or something?

Reply:
Yeah was also not a fan but had nothing better. I like PubliclyExposedConfig.